auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py

```diff
@@ -811,7 +811,7 @@ def parse_input_config(
             for ckpt_key in ["pretrained_ckpt_name", "validate#ckpt_name", "infer#ckpt_name", "finetune#ckpt_name"]:
                 ckpt = override.get(ckpt_key, None)
                 if ckpt and os.path.exists(ckpt):
-                    checkpoint = torch.load(ckpt, map_location="cpu")
+                    checkpoint = torch.load(ckpt, map_location="cpu", weights_only=False)
```

Comment:
I would suggest to add one line to explain why it is False here in case users would read through the script.

Reply:
Sure, added in this commit: https://github.com/Project-MONAI/research-contributions/pull/412/commits/50025344dc9d32818d687596259c9c8c37b9babe